lib/tests/petstore.yaml

```diff
@@ -282,15 +282,15 @@ paths:
         - petstore_auth:
             - write:pets
             - read:pets
-  /pet/{petId}/uploadImage:
+  /pet/{pet-id}/uploadImage:
```

Comment:
i'd rather not touch this file as it's one of the official examples for the openapi yaml, this represent a lof of common use-cases

it's the one that shows in [online swagger editor](https://editor.swagger.io/) by default and I think it's also in the [openapi repo examples](https://github.com/OAI/OpenAPI-Specification/blob/9dff244e5708fbe16e768738f4f17cf3fddf4066/examples/v3.0/petstore.yaml#L57)

Reply:
Thanks for the context, I will create a separate file for the tests